docs/ref/request-response.txt

```diff
@@ -1057,3 +1057,7 @@ otherwise it streams the file out in small chunks.
 
     >>> from django.http import FileResponse
     >>> response = FileResponse(open('myfile.png', 'rb'))
+
+.. warning::
+
+    File object will be closed automatically in FileResponse, so you don't have to open file in a context manager.
```

Comment:
I think it would be more accurate to say that you *can't* use a context manager.

Reply:
Feel free to edit it. :D Should we mention that we can't use a context manager when you pass an object with 'close' attr. Or maybe put the source code on it?